Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -16,43 +18,6 @@ extension PackageShow {
         var title: String
         var url: String
         var score: Int?
-
-        struct History: Equatable {
-            var since: String
-            var commitCount: Link
-            var releaseCount: Link
-        }
-
-        struct Activity: Equatable {
-            var openIssuesCount: Int
-            var openIssues: Link?
-            var openPullRequests: Link?
-            var lastIssueClosedAt: String?
-            var lastPullRequestClosedAt: String?
-        }
-
-        struct ProductCounts: Equatable {
-            var libraries: Int
-            var executables: Int
-        }
-
-        struct ReleaseInfo: Equatable {
-            var stable: DatedLink?
-            var beta: DatedLink?
-            var latest: DatedLink?
-        }
-
-        struct Version: Equatable {
-            var link: Link
-            var swiftVersions: [String]
-            var platforms: [Platform]
-        }
-
-        struct LanguagePlatformInfo: Equatable {
-            var stable: Version?
-            var beta: Version?
-            var latest: Version?
-        }
```

Comment:
I’m moved these to another file but the whole `PackageShow` + model + view part needs some love. I’ve rushed this initially and again for the builds and it feels messy. Not sure if much can be refactored - this kind of data transformation code is just naturally messy - but it could (should!) be more organised.